test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1549,13 +1657,15 @@ def test_cgroup_cpuset_ncpus_are_cores(self):
             self.skipTest('Insufficient information about the processors.')
         if pcpus < 2:
             self.skipTest('This test requires at least two processors.')
-        if sibs/cores == 1:
+        if sibs / cores == 1:
             self.skipTest('This test requires hyperthreading to be enabled.')
         name = 'CGROUP18'
         self.load_config(self.cfg8 % ('', '', '', self.swapctl, ''))
         # Submit M*N jobs, where M is the amount of physical processors and
         # N is number of 'cpu cores' per M. Expect them to run.
         njobs = phys * cores
+        if njobs > 100:
+            self.skipTest("too many jobs (%d) to submit" % njobs)
```

Comment:
Why skip the test in this case? Would it be better to simply put an upper limit on njobs of 100 and let the test run?

Reply:
The test, test_cgroup_cpuset_ncpus_are_cores, tries to submit 1-cpu jobs up to the total number of cpu cores (njobs = pys * cores) available, and then check that an additional job would stay queued due to no more resources. So I can't really put a limit, for it's an absolute limit that is needed to run the test; otherwise, the test will fail for the additional job would still run since resources would still be available.

Reply:
In that case, I'm OK with skipping it. I would suggest you pick a power of two rather than a power of ten as the limit. Maybe 64 or 128?

Reply:
Sure, power of 2.